native/cocos/core/scene-graph/Node.cpp

```diff
@@ -570,13 +569,13 @@ void Node::setWorldRotation(float x, float y, float z, float w) {
 
     _eulerDirty = true;
 
+    notifyLocalRotationUpdated();
```

Comment:
Why invoke it here?

Reply:
Ensure js callback trigger by emit can see the value of new rotation. `setWorldPostion` & `setWorldScale` do act this too.